Content.Shared/CCVar/CCVars.cs

```diff
@@ -31,13 +31,13 @@ public sealed class CCVars : CVars
         /// How long we'll wait until re-sampling nearby objects for ambience.
         /// </summary>
         public static readonly CVarDef<float> AmbientCooldown =
-            CVarDef.Create("ambience.cooldown", 0.5f, CVar.REPLICATED | CVar.SERVER);
+            CVarDef.Create("ambience.cooldown", 0.1f, CVar.REPLICATED | CVar.SERVER);
 
         public static readonly CVarDef<float> AmbientRange =
             CVarDef.Create("ambience.range", 5f, CVar.REPLICATED | CVar.SERVER);
 
         public static readonly CVarDef<int> MaxAmbientSources =
-            CVarDef.Create("ambience.max_sounds", 6, CVar.REPLICATED | CVar.SERVER);
+            CVarDef.Create("ambience.max_sounds", 64, CVar.REPLICATED | CVar.SERVER);
```

Comment:
64 seems like a lot

Reply:
It is a lot, as to allow busy areas with large numbers of ambient entities (say... atmos) to be loud.

Reply:
Just playing this many sounds will cause noticeable framedrops on lower-end systems.

![image](https://user-images.githubusercontent.com/31366439/158065828-d57cb721-72cf-4c18-87fd-062e3e056929.png)

Just from hanging near atmos it becomes the most expensive system.